tox.ini

```
@@ -210,8 +211,10 @@ deps =
     datastore_pymongo-pymongo03: pymongo<4.0
     datastore_pymysql: PyMySQL<0.11
     datastore_pysolr: pysolr<4.0
-    datastore_redis: redis
-    datastore_redis: rb
+    datastore_redis-redislatest: redis
+    datastore_redis-redis040: redis<4.1
```

Let's keep 2 digit number for all versions so this doesn't get confusing.

```suggestion
    datastore_redis-redis0400: redis<4.1
```